nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/RepositoryDescriptor.java

```diff
@@ -310,6 +310,11 @@ public FulltextDescriptor getFulltextDescriptor() {
         return fulltextDescriptor;
     }
 
+    @XNode("indexing/fulltext@fieldSizeLimit")
+    public void setFulltextFieldSizeLimit(int fieldSizeLimit) {
+        fulltextDescriptor.setFulltextFieldSizeLimit(fieldSizeLimit);
+    }
+
```

Comment:
Please add similar changes to `DBSRepositoryDescriptor` using ` @XNode("fulltext@fieldSizeLimit")`.